src/Controllers/Admin/GeneralSettingsController.php

```diff
@@ -27,5 +27,27 @@ public function update( Request $request, Response $response ) {
 		// Add the settings update code here
 
 		return new RedirectResponse( 302, admin_url( 'admin.php?page=dt_launcher&tab=general&updated=true' ) );
+		exit();
+
+	}
+
+	public function update_user_access_settings() {
+		
+		if (isset($_POST['dt_admin_form_nonce']) && wp_verify_nonce($_POST['dt_admin_form_nonce'], 'dt_admin_form')) {
```

Comment:
@Perumal605, please remove the nonce check as it is being handled by middleware.